package.json

```diff
@@ -20,6 +20,7 @@
     "grunt-sass": "^3.1.0",
     "grunt-standard": "^3.2.0",
     "imagemin-cli": "^7.0.0",
+    "npm-run-all": "^4.1.5",
```

Comment:
I can't figure out how to comment on the binary files, but is there a reason for still keeping the old logo?